content/rs/administering/designing-production/security/ldap-integration.md

```diff
@@ -121,9 +121,7 @@ with LDAP authentication in the web UI.
 To create an external user with LDAP authentication in the REST API:
 
 ```src
-$ curl -k -L -v -u "<your_admin_acct>:<your_pword>" --location-trusted \
--H "Content-Type: application/json" -X POST http://<your-res-cluster>:8080/v1/users \
--d "{\"auth_method\": \"external\", \"name\": \"<internal-user-name>\", \"role\": \"<user-role>\"}"
+$ curl -k -L -v -u "<your_admin_account>:<your_password>" -H "Content-Type: application/json" -X POST https://<your_redis_cluster_fqdn>:9443/v1/users -d '{"auth_method":"external", "name":"<username>", "role":"<user_role>"}'
```

Comment:
So we don't use --location-trusted?

Reply:
Location trusted shouldn't be technically needed for this if you are going directly to the https port and there is no redirect. It doesn't add any value to the command that goes already to the encrypted report since there is no redirect.